packages/wonder-blocks-testing/src/fixtures/types.ts

```diff
@@ -16,14 +16,14 @@ export type GetPropsOptions = {
     logHandler: (name: string) => (...args: Array<any>) => void;
 };
 
-export type FixtureProps<TProps extends Record<any, any>> =
+export type FixtureProps<TProps extends object> =
```

Comment:
**note:** This change, along with my update to `flowgen` means that we get the expected `TProps: {...}` output, which ensures that things just work.